wgpu-core/Cargo.toml

```diff
@@ -35,15 +35,15 @@ raw-window-handle = { version = "0.3", optional = true }
 ron = { version = "0.5", optional = true }
 serde = { version = "1.0", features = ["serde_derive"], optional = true }
 smallvec = "1"
-spirv_headers = { version = "1.4.2" }
 thread-id = { version = "3", optional = true }
 tracing = { version = "0.1", default-features = false, features = ["std"] }
 tracing-subscriber =  { version = "0.2", optional = true }
 thiserror = "1"
 
 [dependencies.naga]
 git = "https://github.com/gfx-rs/naga"
-rev = "a9228d2aed38c71388489a95817238ff98198fa3"
+rev = "94802078c3bc5d138f497419ea3e7a869f10916d"
+features = ["spirv"]
```

Comment:
I presume the plan going forward is to gate naga frontends/backends behind features?

Reply:
yes